packages/wallets/src/components/Base/WalletPasswordField/PasswordFieldUtils.ts

```diff
@@ -0,0 +1,50 @@
+export type Score = 0 | 1 | 2 | 3 | 4;
+
+export const passwordPattern = '^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[!-~]{8,25}$';
+
+export const passwordChecker = (password: string) => {
+    const regexList = [
+        {
+            passwordFeedback: 'You should enter 8 - 25 characters.',
+            pattern: /^.{8,25}$/,
+            strength: 1,
+        },
+        {
+            passwordFeedback: 'Password should have lower and uppercase English letters with numbers.',
+            pattern: /[A-Za-z]/,
+            strength: 1,
+        },
+        {
+            passwordFeedback: 'Password should have lower and uppercase English letters with numbers.',
+            pattern: /\d/,
+            strength: 1,
+        },
```

Comment:
```suggestion
        {
            passwordFeedback: 'Password should have lower and uppercase English letters.',
            pattern: /[A-Za-z]/,
            strength: 1,
        },
        {
            passwordFeedback: 'Password should have numbers.',
            pattern: /\d/,
            strength: 1,
        },
```
I think these two needs to be combined or the message needs to change. Refer to the code suggestion.

Reply:
Yes I understand. The reason why I did it as such is because I want to have a different scoring for each of it. But will combine it for sure if you think its the best move 👍🏼

Reply:
Implemented the changes by combining the 2 🙏🏼